webrender/src/resource_cache.rs

```diff
@@ -159,7 +174,7 @@ where
             .cloned()
             .collect::<Vec<_>>();
         for key in resources_to_destroy {
-            self.resources.remove(&key).unwrap();
```

Comment:
why is this removed? I assume we get a warning about unused `Result` or `Option`